detectron2_node:
  ros__parameters:
    detectron_config_file: "COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_3x.yaml"
    detectron_score_thresh: 0.8
    pointcloud2_topic: "/realsense/camera/pointcloud"
    pc_downsample_factor: 16
    min_mask: 20 # minimum mask to be considered as an obstacle candidate
    categories: [0] # please check out COCO dataset category_id list if you want to config this; if you want to track all, leave it empty
    nms_filter: 0.3 # 3D non-max suppression threshold, [0, 1]